sdk/textanalytics/Azure.AI.TextAnalytics/src/MultiLabelClassifyOptions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Options that allow callers to specify details about how the operation
+    /// is run and what information is returned from it by the service.
+    /// </summary>
+    public class MultiLabelClassifyOptions : TextAnalyticsRequestOptions
```

Comment:
It looks like TextAnalyticsRequestOptions includes modelVersion which isn't supported by MultiLabelClassify

Reply:
I noticed that as well, but wasn't sure if we need to "hide" it. I will see if we could skip the derivation and either just define it standalone, or break up the hierarchy a bit. IIRC, Java has completely separate, non-derived options bags classes to avoid this.

Reply:
In java, we try to not extend from TextAnalyticsRequestOptions